mitxpro/templates/base.html

```diff
@@ -57,7 +57,7 @@
     {% block footer %}
         {% include "footer.html" %}
     {% endblock %}
-    {% render_bundle 'django' added_attrs='defer' %}
```

Comment:
We don't want to drop this additional attr

Reply:
Since we don't want to drop the `defer`, we would have to stay with the render_bundle defined in MITxPRO  specifically (renamed to mitol_render_bundle to avoid name collision)   as the `added_attrs` is not defined in ol-django render_bundle. 
we might need to update ol-django-common to add this in render_bundle

Reply:
I'll get that added to `ol-django`

Reply:
@Wassaf-Shahzad if you can review it, I ported that over to ol-django here: https://github.com/mitodl/ol-django/pull/47

Reply:
@Wassaf-Shahzad this is merged and released as `mitol-django-common==2.1.0`. You'll need to update the settings to use the `WEBPACK_*` vars as you noted.